spec/spec_helper.rb

```diff
@@ -28,8 +28,8 @@ def create_page_with_sourcebox iUrl=nil, subcards={},sourcebox=nil
 def create_claim_with_url name,url, subcards={}
   Card::Auth.as_bot do
     sourcepage = create_page url
-    Card.create! :type_id=>Card::ClaimID, :name=>name, 
-                 :subcards=>{ '+source' => {:content=>"[[#{sourcepage.name}]]",:type_id=>Card::PointerID}}.merge(subcards)    
+    Card.create! :type_id=>Card::ClaimID, :name=>name,
+                 :subcards=>{ '+source' => {:content=>"[[#{sourcepage.name}]]",:type_id=>Card::PointerID}}.merge(subcards)
```

Comment:
Line is too long. [122/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.<br>Space inside } missing.<br>Space inside { missing.